Library/Formula/docker-machine.rb

```diff
@@ -3,8 +3,8 @@
 class DockerMachine < Formula
   desc "Create Docker hosts locally and on cloud providers"
   homepage "https://docs.docker.com/machine"
-  url "https://github.com/docker/machine/archive/v0.5.6.tar.gz"
-  sha256 "b6dee30f6e8eb1d3f130cbd9545018e0abb514a968bcd1acd6006ae343f8f402"
+  version "0.5.6"
+  url "https://github.com/docker/machine.git", :revision => "v0.5.6"
```

Comment:
Is there any reason to use the git revision instead of a tarball?

If so, please use `:tag` for the tag and `:revision` for the actual commit hash, e.g.:
```rb
  url "https://github.com/docker/machine.git",
  :tag => "v0.5.6", :revision => "the-commit-hash"
```